app/Helpers/ApiHelper.php

```diff
@@ -23,11 +23,15 @@ public static function checkUserApiQuota($username) {
             $api_quota = env('SETTING_ANON_API_QUOTA') ?: 5;
         }
 
+        if ($api_quota == -1) {
```

Comment:
Let's use `$api_quota < 0` to ensure that the PR doesn't go against published API documentation :)